src/hotspot/os/linux/cgroupV1Subsystem_linux.hpp

```diff
@@ -51,50 +50,67 @@ class CgroupV1Controller: public CgroupController {
     char *subsystem_path() { return _path; }
 };
 
-class CgroupV1MemoryController: public CgroupV1Controller {
+class CgroupV1MemoryController : public CgroupMemoryController {
```

Comment:
Do we expect this class to be subclassed later on/in the future? If not, then we can declare it final as so:

```c++
class CgroupV1MemoryController final : public CgroupMemoryController
```